zio-http/jvm/src/main/scala/zio/http/netty/server/NettyDriver.scala

```diff
@@ -77,7 +77,7 @@ private[zio] final case class NettyDriver(
   override def toString: String = s"NettyDriver($serverConfig)"
 }
 
-private[zio] object NettyDriver {
+object NettyDriver {
```

Comment:
Review reminder: I made the NettyDriver public to make it usable for anyone who wants to utilize it when writing tests.